roseau/load_flow/models/lines/lines.py

```diff
@@ -72,6 +73,10 @@ def __init__(
 
             geometry:
                 The geometry of the line i.e. the linestring.
+
+            max_loading:
```

Comment:
Say that it is not used in the load flow (as we do for all other optional parameters)